src/api/version.ts

```diff
@@ -8,14 +8,27 @@
  * SPDX-License-Identifier: EPL-2.0
  **********************************************************************/
 
+import axios from 'axios'
 import execa = require('execa')
+import * as fs from 'fs'
+import * as https from 'https'
 import Listr = require('listr')
+import * as path from 'path'
 
 import { CheTasks } from '../tasks/che'
-import { getClusterClientCommand } from '../util'
+import { getClusterClientCommand, getProjectName, getProjectVersion } from '../util'
 
 import { KubeHelper } from './kube'
 
+const CHECTL_DEVELOPMENT_VERSION = '0.0.2'
```

Comment:
?

Reply:
This is hardcoded version which is always used when `chectl` is built form source. So I'm considering it as a nightly (such behaviour will not break the behaviour while developing chectl)